lib/ActivityStreams.js

```diff
@@ -310,13 +317,6 @@ ActivityStreams.prototype = {
     this.send(am.actions.Response("EXPERIMENTS_RESPONSE", this._experimentProvider.data), worker);
   },
 
-  _respondToPrefsRequest() {
```

Comment:
I can't add the comment where I *really* want to 😞 , `ActivityStreams.js:355` calls this function which was removed.